Meta/run.py

```diff
@@ -898,8 +905,7 @@ def configure_and_run():
 
     arguments = assemble_arguments(config)
 
-    build_directory = environ.get("SERENITY_BUILD", ".")
-    os.chdir(build_directory)
+    os.chdir(BUILD_DIRECTORY)
```

Comment:
If you have time to put that fix in its own commit, that would be nice :^)

Reply:
Doesn't look like this is done?

Reply:
I moved the SERENITY_BUILD_DIR environment variable fix out of 730cfbd1f1b5fde5ca7c9d6508b556246d87e361 into a new commit aea8ff48219db53f206b6b85d680fe99eb838fe5.
Or was that review comment referring to something else?

Reply:
Oh sorry, I thought Lucas meant removing the `build_directory` local. Maybe I misunderstood what he meant, though!

Reply:
My bad it wasn't explicit, but fixing the name was my main concern. I don't mind where the other change ends up.